modules/tinymce/src/core/main/ts/init/InitContentBody.ts

```diff
@@ -219,6 +219,7 @@ const autoFocus = (editor: Editor) => {
 
       if (!focusEditor.destroyed) {
         focusEditor.focus();
+        focusEditor.getBody().scrollIntoView();
```

Comment:
Another issue is that `scrollIntoView` does not work very well when the editor is already in view... it can often cause it to scroll to the top. For example, try https://fiddle.tiny.cloud/ffiaab/7 and click the focus second button. Even though the second editor is in view it causes it to scroll to the top.

As such, we probably want to consider using our `ScrollIntoView` logic instead which accounts for this case.

- [ ] Fix this issue and add a test to cover it

Reply:
I'm fairly sure we wrote some custom scroll code for TBIO to provide an approximation of the 'if needed' code from Safari on all browsers. I wonder if we could dig that up. Sadly I don't really remember where to look.

Reply:
The `ScrollIntoView` module I mentioned above was rewritten back around 5.1 based on the original TBIO logic.

Reply:
ah I think I misread what you said. Yes that's exactly what we need.

- [ ] use ScrollIntoView logic